index.html

```diff
@@ -0,0 +1,69 @@
+    </h1>
+    </nav>
+</header>
+<section id="main">
+    <iframe width="750" height="440" src="https://www.youtube.com/embed/XX-Kct0PfFc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <p>Git & GitHub Tutorial for Beginners #9 - Merging Branches (& conflicts)</p>
+    
+    <p>329,583 viewsMay 13, 2016</p>
+</section>
+    <aside id="watching_youtube" class="youtuber">
+        <div class="archor">
+            <ul class="archor-ul">
+                <li class="archor-ul__li">
+                    <a href="#">All</a>
+                </li>
+                <li class="archor-ul__li">
+                    <a href="#">Related</a>
+                </li>
+                <li class="archor-ul__li">
+                    <a href="#">Watched</a>
+                </li>
+                <li class="archor-ul__li">
+                    <a href="#">All about JavaScript</a>
+                </li>
+            </ul>
+        </div>
+    <iframe class="watch" width="560" height="315" src="https://www.youtube.com/embed/hdI2bqOjy3c" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <ul>
+        <li>JavaScript Crash Course For Beginners</li>
+    </ul>
+    <iframe class="watch" width="560" height="315" src="https://www.youtube.com/embed/PkZNo7MFNFg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <ul>
+        <li>Learn JavaScript-Full Course for Beginners</li>
+    </ul>
+    <iframe class="watch" width="560" height="315" src="https://www.youtube.com/embed/W6NZfCO5SIk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <ul>
+        <li>JavaScript Course for Beginners</li>
+    </ul>
+    <iframe class="watch" width="560" height="315" src="https://www.youtube.com/embed/PlxWf493en4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <ul>
+        <li>JavaScript Tutorial for Beginners</li>
+    </ul>
+    <iframe class="watch" width="560" height="315" src="https://www.youtube.com/embed/0afZj1G0BIE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <ul>
```

Comment:
I proppose you to try and use just a single UL element so you could cover the whole section and you just add a few properties in your css.